ci/cppclean.sh

```diff
@@ -42,7 +42,7 @@ fi
 if [ ${NO_WARNINGS_DIRECT} -gt 19 ]; then
     errors+="Failed probably due to not using static keyword with functions definitions: ${NO_WARNINGS_DIRECT}"$'\n'
 fi
-if [ ${NO_WARNINGS_NOTUSED} -gt 4 ]; then
+if [ ${NO_WARNINGS_NOTUSED} -gt 6 ]; then
```

Comment:
Why cant we just forward declare?

Reply:
https://github.com/openvinotoolkit/model_server/pull/2177/commits/4b8882481f1ed64108e2a2909524288abb77349a